src/Facility.Definition/ServiceDefinitionUtility.cs

```diff
@@ -95,7 +95,8 @@ internal static IEnumerable<ServiceDefinitionError> ValidateName(string name, Na
 
 		internal static IEnumerable<ServiceDefinitionError> ValidateTypeName(string name, NamedTextPosition position)
 		{
-			if (ServiceTypeInfo.TryParse(name, x => s_validNameRegex.IsMatch(x) ? new ServiceDtoInfo(x) : null, position, out ServiceDefinitionError error) == null)
+			ServiceDefinitionError error;
+			if (ServiceTypeInfo.TryParse(name, x => s_validNameRegex.IsMatch(x) ? new ServiceDtoInfo(x) : null, position, out error) == null)
```

Comment:
I am eager to upgrade the compiler, build environment, publishing strategy, etc. to something like https://github.com/Faithlife/Parsing but haven't yet found the time.